websites/docs-smoke-test/src/code-examples/example.php

```diff
@@ -0,0 +1,16 @@
+public class MyClass {
+  public static void main(String[] args) {
+    System.out.println("Hello World");
+  }
+}
```

Comment:
Bad copy-paste?

Reply:
https://gph.is/2pbhdJ3

Reply:
(face-palm)

Reply:
This is strange, this doesn't exist for me locally.